config/crds/v1/all-crds.yaml

```
@@ -3724,6 +3798,15 @@ spec:
                 description: ElasticsearchHealth is the health of the cluster as returned
                   by the health API.
                 type: string
+              monitoringAssociationStatus:
+                additionalProperties:
+                  description: AssociationStatus is the status of an association resource.
+                  type: string
+                description: AssociationStatusMap is the map of association's namespaced
+                  name string to its AssociationStatus. For resources that have a
+                  single Association of a given type (eg. single ES reference), this
+                  map will contain a single entry.
```

```suggestion
                description: AssociationStatusMap is the map of association's namespaced
                  name string to its AssociationStatus. For resources that have a
                  single Association of a given type (for ex. single ES reference), this
                  map contains a single entry.
```
This sentence is bit unclear:
AssociationStatusMap is the map of association's namespaced name string 
Not sure how to reword it...